main.py

```diff
@@ -298,6 +298,20 @@ async def clear_scores() -> None:
     print(f"Deleted all clearable scores in {time.time() - start_time:.2f} seconds")
 
 
+async def update_hanayo_country_list() -> None:
+    COUNTRY_LIST_KEY = "hanayo:country_list"
+    
+    print("Updating hanayo country list")
+    start_time = int(time.time())
+
+    country_list = await db.fetchall("SELECT DISTINCT(country) FROM users WHERE LENGTH(country) = 2 AND country != 'XX'")
+    async with redis.pipeline() as pipe:
+        await pipe.del(COUNTRY_LIST_KEY)
+        await pipe.zadd(COUNTRY_LIST_KEY, {country["country"]: 0 for country in country_list})
```

Comment:
what's the 0 val for?

Reply:
nothing, really. hanayo just expects a sorted set (it calls zrevrange (maybe this could be used to sort the country selector by pp or something))

Reply:
should they have real values? is it the sum of country pp or something? or always 0 in redis live rn?

Reply:
it used to be populated with the amount of ranked users in a country, should be fairly easy to include here

Reply:
> it used to be populated with the amount of ranked users in a country

I suspect it's used for the ordering of countries displayed in the list

Reply:
yeah it is